napari/components/_interaction_box_mouse_bindings.py

```diff
@@ -183,20 +183,23 @@ def _on_mode_change(self, event):
     def _on_transform_change(self, event):
         """Gets called when the interaction box is transformed to update transform of the layer"""
 
+        viewer = self._ref_viewer()
+        active_layer = viewer.layers.selection.active
+        if active_layer is None:
+            return
+
         layer_dims_displayed = dims_displayed_world_to_layer(
             list(self._ref_viewer().dims.displayed),
-            self._ref_viewer().dims.ndim,
-            self._ref_viewer().layers.selection.active.ndim,
+            viewer.dims.ndim,
+            active_layer.ndim if active_layer is not None else 2,
```

Comment:
```suggestion
            active_layer.ndim,
```

Given the early return above, I think this ternary statement will always evaluate to `active_layer.ndim`. That allows us to remove the magic number `2` here.